src/services/legal-services.ts

```diff
@@ -236,7 +244,16 @@ export default class LegalServices {
       .then(response => {
         const parties = response?.data?.parties
         if (parties) {
-          return parties
+          // WORK-AROUND WARNING !!!
+          // convert parties from "middleInitial" to "middleName"
+          return parties.map(party => {
+            const middleInitial = party.officer['middleInitial']
+            if (middleInitial !== undefined) {
+              party.officer.middleName = middleInitial
+              delete party.officer['middleInitial']
+            }
+            return party
+          })
```

Comment:
It turns out this workaround is needed for Change / Conversion / CoopCorrection / CorpCorrection / FirmCorrection filings.

It was already implemented for Change and Conversion so I removed it from there.

Reply:
Sample change:

![image](https://github.com/bcgov/business-edit-ui/assets/10002889/ac723dce-b9c6-474e-ad57-93cca84d8d67)

Reply:
Sample conversion:

![image](https://github.com/bcgov/business-edit-ui/assets/10002889/d5b7b1e3-19b0-4bcd-8217-ae9025fef571)

Reply:
Sample coop correction:

![image](https://github.com/bcgov/business-edit-ui/assets/10002889/e7a215b2-6d91-4de3-9b3d-d7314c838562)

Reply:
Sample firm correction:

![image](https://github.com/bcgov/business-edit-ui/assets/10002889/5d1adeb3-30bc-4e91-8895-e6c872c6deb1)